content/departments/product-engineering/engineering/cloud/delivery/on-call.md

```diff
@@ -0,0 +1,126 @@
+
+This document is designed to help the Incoming Engineer On-Call get up to speed with any open issues that require their attention during their shift. Additionally, this is designed to provide a mechanism for the Outgoing Engineer On-Call to
+offload support duties in order to transition back into project work.
+
+### Ensure OpsGenie can page you
+
+This can be done in the OpsGenie interface by supplying your cellphone number or by downloading the Opsgenie app.
+The method you choose to be contacted is up to you as long as your response time follows the [Reponse Time guidelines](#response-times-and-slas)
+
+### Join Slack Channels
+
+The following Slack channels should be monitored during your shift. There is no expectation to actively monitor these channels 24/7 during your support week, but you should serve as the designated point of contact during your working hours.
+
+| Channel Name              | Description                                                                                                    |
+| ------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| #delivery                 | Public channel where Soucegraph teammates can ask questions, provide information, and escalate support issues. |
+| #alerts-managed-instances | Application-level webhook alerts from Managed Instances.                                                       |
+| #incidents                | Company-wide incident feed.                                                                                    |
+| #prod-eng-announcements   | Product Engineerng Announcements.                                                                              |
+| #ask-prod-eng             | Company-wide forum to discuss Product Engineering topics. Generally questions are tagged with "[Delivery]"     |
+| #buildkite-main           | Information about BuildKite CI.                                                                                |
+
+|
+
+#### The `@delivery-support` Slack handle
+
+You will be automatically added to the `@delivery-support` Slack group by Opsgenie. This handle is used to call attention to
+important topics in Slack that may or may not include incidents or P1 customer support issues.
+
+While an immediate response is not necessarily required, use your best judgement to determine an appropriate response.
+
+## On-Call Duties
+
+This is section is not designed to be prescriptive. You should use your best judgement on how to handle your on-call duties. This is by no means an exhaustive list but some notes of things to check while on-duty.
+
+### Monitoring Slack Channels
+
+You are not expected to monitor slack 24/7. During your working hours periodically check for messages in the
```

Comment:
Additionally - delivery engineers who are _not_ on call do not need to respond to questions. We should get in the habit of letting the on-call engineer provide a first reply, so that we get more focus time and everyone gets practiced at responding. (There's a tendency for a SME to become the only one answering certain types of questions, and this will force everyone to get exposed to those answers).